docs/src/main/tut/datatypes/functionk.md

```diff
@@ -121,3 +121,24 @@ type ErrorOr[A] = Either[String, A]
 val errorOrFirst: FunctionK[List, ErrorOr] =
   λ[FunctionK[List, ErrorOr]](_.headOption.toRight("ERROR: the list was empty!"))
 ```
+
+## Natural Transformation
+
+In category theory, a [Natural Transformation](https://en.wikipedia.org/wiki/Natural_transformation) provides a morphism between Functors while preserving the internal structure. It's one of the most fundamental notions of category theory.
```

Comment:
ncatlab might be a better reference than Wikipedia for such a subject :-) https://ncatlab.org/nlab/show/natural+transformation